WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
 			}
 			is_adult = yes
-			prestige = 5000
+			prestige = 2000
```

Comment:
> Please stop playing the moral high ground and pretending you are innocent in all this.

No one is guilty, so per definition I can't be innocent either.

> You don't subject anyone to half the scrutiny I get.

I read through all code. Difficult to do it more or less. 😛 

> I'd work with anyone else on this Github compiliation if I could, but I'm stuck with you, because no one else is within contact.

I would still read through all code.